docs/examples/basic_tracer/README.rst

```diff
@@ -0,0 +1,37 @@
+Basic Trace
+===========
+
+These examples show how to use OpenTelemetry create and export Spans.
+
+There are two different examples:
+
+* basic_trace: Shows to how to configure a SpanProcessor and Exporter, and how to create a tracer and span.
+
+* resources: Shows how to add resource information to a Provider. Note that this must be run on a GCP instance.
+
+The source files of these examples are available :scm_web:`here <docs/examples/basic_tracer/>`.
```

Comment:
When I view the rich diff I see colons and backticks here, not the formatting you intend. Is that a limitation of GitHub, or do you need to format this differently?

Reply:
Seems these README files can't be viewed on Github, (similar thing happens here https://github.com/open-telemetry/opentelemetry-python/tree/master/docs/examples/basic_meter) when I build the files locally it works so we should be good :)